.github/workflows/ci.yml

```diff
@@ -38,6 +38,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - uses: wyvox/action-setup-pnpm@v3
+        with: { node-version: 22 }
```

Comment:
why specify this? 🤔 

we could add an `.nvmrc` or `.node-version`, perhaps?

Reply:
To get `Promise.withResolvers()`.

It's of course only a few lines to do what `withResolvers()` does, but `withResolvers()` is so nice and I figured this library is pretty cutting edge, so...

Reply:
oh yes, I mean, setting node 22 is great!

Will also need to update:
- https://github.com/proposal-signals/signal-utils/blob/main/package.json#L83